common/script/index.js

```diff
@@ -284,23 +280,14 @@ api.wrap = function wrapUser (user, main = true) {
     randomDrop: _.partial(importedFns.randomDrop, user),
     autoAllocate: _.partial(importedFns.autoAllocate, user),
     updateStats: _.partial(importedFns.updateStats, user),
+    statsComputed: _.partial(importedFns.statsComputed, user),
     ultimateGear: _.partial(importedFns.ultimateGear, user),
     nullify: _.partial(importedFns.nullify, user),
   };
 
   Object.defineProperty(user, '_statsComputed', {
     get () {
-      let computed = _.reduce(['per', 'con', 'str', 'int'], (m, stat) => {
-        m[stat] = _.reduce($w('stats stats.buffs items.gear.equipped.weapon items.gear.equipped.armor items.gear.equipped.head items.gear.equipped.shield'), (m2, path) => {
-          let item;
-          let val = user.fns.dotGet(path);
-          return m2 + (path.indexOf('items.gear') !== -1 ? (item = content.gear.flat[val], (Number(item ? item[stat] : undefined) || 0) * ((item ? item.klass : undefined) === user.stats.class || (item ? item.specialClass : undefined) === user.stats.class ? 1.5 : 1)) : Number(val[stat]) || 0);
-        }, 0);
-        m[stat] += Math.floor(api.capByLevel(user.stats.lvl) / 2);
-        return m;
-      }, {});
-      computed.maxMP = computed.int * 2 + 30;
-      return computed;
+      return user.fns.statsComputed();
```

Comment:
I think he means that the statsComputed file should be moved to /libs, imported here and called like `statsComputed(user)` - when not using user.{fns|ops} you should always pass `user`